eftcamb/06p1_abstract_EFTCAMB_full_map.f90

```diff
@@ -84,8 +85,14 @@ subroutine EFTCAMBFullModelComputeAdotoa( self, a, eft_par_cache, eft_cache )
         type(EFTCAMB_parameter_cache), intent(inout) :: eft_par_cache !< the EFTCAMB parameter cache that contains all the physical parameters.
         type(EFTCAMB_timestep_cache ), intent(inout) :: eft_cache     !< the EFTCAMB timestep cache that contains all the physical values.
 
-        write(*,*) 'IW'
-        stop
+        real(dl)    :: temp
+
+        temp = 1.0_dl/(1.0_dl + eft_cache%EFTOmegaV+ a*eft_cache%EFTOmegaP)*(eft_cache%grhom_t+ 2.0_dl*eft_cache%EFTc -eft_cache%EFTLambda )/3.0_dl
```

Comment:
Remove check to temp as it will slow down the code. If this happens the code will throw errors in other places